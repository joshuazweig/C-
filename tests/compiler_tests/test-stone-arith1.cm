int main () {
    stone s1;
    stone s2;
    stone s3;
    stone s4;
    stone s5;

    s1 = "7";
    s2 = "13";
    s3 = "16";
    s4 = "20";
    s5 = "123";

    print_stone(s1);
    print_stone(s2);
    print_stone(s3);
    print_stone(s4);
    print_stone(s5);
}

