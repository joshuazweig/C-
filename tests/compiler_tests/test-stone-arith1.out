7
D
10
14
7B
